Makefile

```diff
@@ -36,13 +36,21 @@ user-manual-de-pdf:
 	@echo "User manual de build finished; PDF is updated"
 
 api-docs: clean-api-docs
-	cd build && sh get-server-sources.sh master
+	cd build && sh get-server-sources.sh $(DRONE_BRANCH)
```

Comment:
As we are not building this on drone for the live system we need to add the env variable there as well. Let me do that.

Reply:
Okay :)
Thanks a lot Morris!